docs/Customization/Memory-Hierarchy.rst

```diff
@@ -16,6 +16,12 @@ configure 4 KiB direct-mapped caches for L1I and L1D.
 
     import freechips.rocketchip.subsystem.{WithNMediumCores, WithNSmallCores}
```

Comment:
I would delete this line and instead have these configs have the full package path:
```
new freechips.rocketchip.subsystem.XYZMixin(...) ++
```